src/foam/nanos/om/OM.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.om',
+  name: 'OM',
+
+  documentation: `An Operational Measure which captures the count of some event.`,
+
+  javaImports: [
+    'foam.core.X'
+  ],
+
+  properties: [
+    {
+      class: 'Class',
+      name: 'classType'
+    },
+    {
+      class: 'String',
+      name: 'name'
+    },
+    {
+      name: 'created',
```

Comment:
Could implement `CreatedAware` and use `CreatedAwareDAO` instead.

Reply:
Yes, since we put OMs immediately, CreatedAwareDAO will work,  was in the PM mindset where we create but not put until later.

Reply:
Going to leave this out for now as it adds complexity on the proxy delegate replacement on rollup.  When we decide which way we are going - this or Candlestick then we can make use of CreatedAware.